### ISSUE TITLE
AttributeError: 'SVR' object has no attribute '_probA'

### ISSUE THREAD

**Issue by Jiajie-Ye** [buyizhiyou/NRVQA#5]
When I run `python test.py --mode brisque --path=imgs/origin.jpeg`
I got this error.
Am I missing some dependencies?
Please share the dependencies list on conda or pip.
Thank you very much.

**Comment by buyizhiyou** [comment]
> When I run `python test.py --mode brisque --path=imgs/origin.jpeg`
> I got this error.
> Am I missing some dependencies?
> Please share the dependencies list on conda or pip.
> Thank you very much.

I run successfully in the follow environment;
python3
opencv3.4.2
joblib==0.14.0
numpy==1.17.4
scikit-learn==0.22
scipy==1.2.1

**Comment by Jiajie-Ye** [comment]
Thank you very much
And I find that after compare the results of original image and compression image, their quality scores are similar
imgs/origin.jpeg-----brisque score:0.4922963373236703
imgs/compression.jpeg-----brisque score:0.4922910845354642
Are the results correct?

**Comment by buyizhiyou** [comment]
> Thank you very much
> And I find that after compare the results of original image and compression image, their quality scores are similar
> imgs/origin.jpeg-----brisque score:0.4922963373236703
> imgs/compression.jpeg-----brisque score:0.4922910845354642
> Are the results correct?

yes,you are right!The trained brisque model  not preforms well ! you can train a new model in your own dataset!
